orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -633,9 +635,10 @@ def test_bert_inputs_with_dynamic_shape():
         x, y, z = _get_bert_for_sequence_classification_sample_data_with_random_shapes('cuda')
 
         outputs = model(x, y, None, None, None, None, z)
-        s = outputs[0]
-        s.backward()
+        loss = outputs[0]
+        loss.backward()
 
+        assert loss is not None
```

Comment:
The output for loss varies a lot even with same inputs. Tried even 1e-1 for atol param, the diff could be out of the range sometimes. It seems that there's something else random in the bert model, thus can't apply torch.allclose() or _test_helpers.assert_gradients_match_and_reset_gradient(). THerefore I am adding simple check just to ensure dynamic shape inputs would work.

Reply:
are you comparing loss to a fixed value? there is some dropout op in the bert model. If you wish to compare it to a fixed value, you would need to fix a seed on the random value. try torch.manual_seed()

But I would be interested to compare the loss produced by a pytorch run. Is that possible ?

Reply:
No, I was comparing loss between ortmodule and pytorch run, let me try torch.manual_seed() and see if that helps, thx.

Reply:
Still unable to produce tow closed loss values, checking around ...